client/me/purchases/remove-purchase/remove-domain-dialog/index.jsx

```diff
@@ -48,6 +52,11 @@ class RemoveDomainDialog extends Component {
 						components: { strong: <strong /> },
 					}
 				) }
+				{ hasTitanWithUs &&
+					translate(
+						' You also have an active Titan Mail subscription for this domain, and your email will stop ' +
+							'working if you delete your domain.'
```

Comment:
Super minor but there is white space at the beginning of this string. I would also suggest using plural when referring to emails since a user can have several mailboxes. That may just be personal preference but we could also make `stop working` stands:

> You also have an active Titan Mail subscription for this domain, and your emails will **stop working** if you delete your domain.